src/styles/content/tables/table-cell-inactive/styles.less

```diff
@@ -0,0 +1,131 @@
+& when (@table-cell-inactive-enabled) {
+  /* Table */
+
+  .table {
+
+    tr.inactive td,
+    td.inactive {
+      .element-text-style(table-cell-inactive);
```

Comment:
I'm in favor of the suggestion. The intention was to put this code in CNVS so in that case it would have to follow the best practices of CNVS. Making this change means we keep this code and maintain it ourselves.